app/models/data.js

```diff
@@ -52,4 +52,26 @@ Data.prototype.delete = function (key, callback) {
   });
 };
 
+Data.prototype.deleteOlderThanDate = function (date, callback) {
+  this.db.find({}, (err, docs) => {
+    if (err) {
+      return callback(err);
+    }
+
+    if (docs.length > 0) {
+      docs.forEach((item)=>{
+        let updatedAtDate = new Date(item.updatedAt);
+        if (updatedAtDate.getTime() < date) {
```

Comment:
Okay.  It's just unfortunate that we're using a database and not using the database features.